JavaAssignment/src/akshata/Assignment_Q10.java

```diff
@@ -0,0 +1,31 @@
+package akshata;
+
+public class Assignment_Q10 {
+
+	static int sum;
+
+	public static void main(String[] args) {
+
+		int num = Integer.parseInt(args[0]);
+		System.out.println("user given no is  :" + num);
+		for (int i = 2; i < num; i++) {
```

Comment:
you can separate this lines of logic in other methods, so the main method will only do calling a job.